account_invoice_comment_template/__manifest__.py

```diff
@@ -2,19 +2,21 @@
 # Copyright 2013-2014 Nicolas Bessi (Camptocamp SA)
 # Copyright 2018 Qubiq - Xavier Jiménez
 # Copyright 2018 Tecnativa - Pedro M. Baeza
+# Copyright 2021 Tecnativa - Víctor Martínez
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
 {
-    "name": "Account Invoice Comments",
+    "name": "Account Invoice Comments Template",
     "summary": "Comments templates on invoice documents",
-    "version": "12.0.2.0.0",
-    "category": "Sale",
-    "author": "Camptocamp, " "Tecnativa, " "Odoo Community Association (OCA)",
+    "version": "13.0.1.0.0",
+    "category": "Accounting & Finance",
+    "author": "Camptocamp, Tecnativa, Odoo Community Association (OCA)",
+    "development_status": "Mature",
     "license": "AGPL-3",
     "installable": True,
-    "depends": ["account", "base_comment_template",],
+    "depends": ["l10n_generic_coa", "base_comment_template"],
```

Comment:
This dependency is not correct. We must do tests that don't force us to install a specific CoA.